_pages/hcd2ha.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: hcd2ha
+pageUrl: hcd2ha.github.io
+timestamp: 2017-01-25
+---
```

Comment:
@hectorsector What do you mean by that?
pressing Enter? If thats the case, 6th line is added.